apps/prairielearn/src/pages/newsItem/newsItem.html.ts

```diff
@@ -51,16 +51,14 @@ export function NewsItem({
             </header>
 
             ${unsafeHtml(newsItemHtml)}
-
-            <aside>
-              <p class="text-right small border-top mt-5 pt-3">
-                Want to help make PrairieLearn better? It's open source and contributions are
-                welcome
-                <a href="https://github.com/PrairieLearn/PrairieLearn" target="_blank">on GitHub</a
-                >!
-              </p>
-            </aside>
           </article>
+
+          <aside>
+            <p class="text-right small border-top mt-5 pt-3">
+              Want to help make PrairieLearn better? It's open source and contributions are welcome
+              <a href="https://github.com/PrairieLearn/PrairieLearn" target="_blank">on GitHub</a>!
+            </p>
+          </aside>
```

Comment:
This change fixed a failing accessibility test that was introduced by upgrading `axe-core`.